### PR TITLE
fix #13

### LINKED ISSUE #13: Constant value never working
Hello I was following the examples to validate against a date range

#### Schema

```json
{
    "definitions": {},
    "$schema": "http://json-schema.org/draft-07/schema#",
    "$id": "https://some-url/mini-schema.json",
    "type": "object",
    "title": "Mini Schema",
    "required": [
        "year"
    ],
    "additionalProperties": false,
    "properties": {
        "year": {
            "$id": "#/properties/year",
            "type": "string",
            "title": "Year (4 digits)",
            "description": "a valid year string in YYYY format",
            "default": "",
            "moment": {
                "format": [
                    "YYYY"
                ],
                "validate": [
                    {
                        "test": "isSameOrAfter",
                        "value": "1900-01-01T00:00:00.000Z"
                    },
                    {
                        "test": "isSameOrBefore",
                        "value": "2999-01-01T00:00:00.000Z"
                    }
                ]
            },
            "examples": [
                "2020"
            ]
        }
    }
}
```

#### Data

```js
{
    year: '2010'
}
```

#### Error text

```bash
data/year "isAfter" validation failed for value(s): null
```

### Library Versions

```
    "ajv": "^6.10.2",
    "ajv-keywords": "^3.4.1",
    "ajv-moment": "^2.0.3",
    "moment": "^2.24.0",
    "moment-timezone": "^0.5.27"
```

I think the problem is at this line of code https://github.com/cludden/ajv-moment/blob/master/lib/index.js#L82

```js
const _val = { now: val.now, $data: val.$data, manipulate: val.manipulate };
            if (_val.now !== true && typeof _val.$data === 'string') {
                _val.value = it.util.getData(_val.$data, it.dataLevel, it.dataPathArr);
            }
```

`_val.value` is always `undefined` when "value" is a constant like in my example

Should be

```js
const _val = { now: val.now, $data: val.$data, manipulate: val.manipulate };
 if (typeof val === 'string'){
    _val.value = JSON.stringify(val);
 }
else  if (_val.now !== true && typeof _val.$data === 'string') {
   _val.value = it.util.getData(_val.$data, it.dataLevel, it.dataPathArr);
}
```

I will try to submit a PR

### PR DESCRIPTION
**Pull request by lamuertepeluda**
Hi there,

I wrote a fix for #13 

Let me know if everything's alright :)

Cheers

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -80,7 +80,10 @@ function inline(it, keyword, schema) {
         }
         _validation.value = _validation.value.map(function (val) {
             const _val = { now: val.now, $data: val.$data, manipulate: val.manipulate };
```
Comment (cludden) [inline lib/index.js]: does this line throw when `val` is a string?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ajv-moment",
-  "version": "2.0.3",
+  "version": "2.0.4",
```
Comment (cludden) [inline package.json]: can you leave this out? releases are handled outside the PR process

File: lib/index.js (hunk 1 above)
Comment (lamuertepeluda) [reply lib/index.js]: Screenshot_20200207-230358~2

Not at all! :) See the attached screenshot (sorry I am from my mobile away from the PC right now).
The reason is that strings are basically objects. You will get all values set to undefined, but safe enough.

File: package.json (hunk 2 above)
Comment (lamuertepeluda) [reply package.json]: ok I will change this

Comment (lamuertepeluda) [reply package.json]: hi, it should be ok now :)

**Pull request merged**